lib/tasks/email_tos_update.rake

```diff
@@ -0,0 +1,18 @@
+namespace :email do
+  task :tos_update => :environment do
+    puts "Emailing #{Publisher.count} users"
+
+    Publisher.find_each.with_index do |user, index|
```

Comment:
I believe we're only emailing active publishers

Reply:
Mandar mentioned that "- we send it to everyone - including suspended and in setup (just no deletes or erasures)"